python/lsst/cp/pipe/linearity.py

```diff
@@ -181,9 +181,9 @@ def run(self, inputPtc, dummy, camera, inputDims):
         -----
         This task currently fits only polynomial-defined corrections,
         where the correction coefficients are defined such that:
-            corrImage = uncorrImage + sum_i c_i uncorrImage^(2 + i)
+        corrImage = uncorrImage + sum_i c_i uncorrImage^(2 + i)
         These `c_i` are defined in terms of the direct polynomial fit:
-            meanVector ~ P(x=timeVector) = sum_j k_j x^j
+        meanVector ~ P(x=timeVector) = sum_j k_j x^j
         such that c_(j-2) = -k_j/(k_1^j) in units of DN^(1-j) (c.f.,
         Eq. 37 of 2003.05978). The `config.polynomialOrder` or
         `config.splineKnots` define the maximum order of x^j to fit.
```

Comment:
I think I formatted this correctly, but I don't think it shows in my browser; it just adds \( and \) around the math regions.

Reply:
I asked about it on Slack. Note that if the display error gets fixed, there will still be other problems (e.g., `sum` instead of `\sum`), though I'm not sure if there's any point addressing them before that.